qa/suites/rados/thrash-old-clients/ceph.yaml

```
@@ -1,2 +1,3 @@
+verify_ceph_hash: false
```

```suggestion
verify_ceph_hash: false  # Don't verify os + flavor + sha1
```

I found the name `hash` a bit misleading.

---

yes, "hash" is more related to how this function is implemented -- it query shaman with the sha1 of the ceph being tested for its packages. and as you put sha1 is only one of the parameters of the query. will incorporate your change.

---

also, i want the name align with its counterpart "suite_verify_ceph_hash" in teuthology. so one can grep / search it for more context in future.

---

updated and repushed.